dataset-spec.html

```diff
@@ -347,5 +338,23 @@ <h3><dfn>QuadRunIteratee</dfn> interface</h3>
     <p>A callable function that can be executed on a quad.</p>
   </section>
 </section>
+
+<section>
+  <h2>Runtime Semantics</h2>
+
+  <section>
+    <h3>Quad Matching</h3>
+
+    <p>The methods <code>match</code> and <code>deleteMatches</code> select quads using on the following logic:</p>
+    <ul>
+      <li>calling <code>quad.subject.equals</code> with the specified subject as an argument returns true, or the subject argument is null, AND</li>
+      <li>calling <code>quad.predicate.equals</code> with the specified predicate as an argument returns true, or the predicate argument is null, AND</li>
+      <li>calling <code>quad.object.equals</code> with the specified object as an argument returns true, or the object argument is null, AND</li>
+      <li>calling <code>quad.graph.equals</code> with the specified graph as an argument returns true, or the graph argument is null</li>
+    </ul>
+
+    <p>The rules are applied with AND functionality, so only quads matching all of the given non-null arguments will be selected.</p>
```

Comment:
```suggestion
    <p>Only quads matching all of the given non-null arguments will be selected.</p>
```

I would remove the first part of this sentence, *AND functionality* is confusing to me, the listed algo is pretty clear with the `AND` at each step, talking about *AND functionality* makes it less clear IMO.